src/source/Metrics/Metrics.h

```diff
@@ -25,10 +23,11 @@ STATUS getIceCandidatePairStats(PRtcPeerConnection, PRtcIceCandidatePairStats);
  * @brief Get specific ICE candidate stats
  * @param [in] PRtcPeerConnection Contains the Ice agent object with diagnostics object
  * @param [in/out] PRtcIceCandidateStats Fill up the ICE candidate stats for application consumption
+ * @param [in] BOOL If TRUE, local candidate stats are extracted, else remote candidate stats are extracted
  * @return Pass/Fail
  *
  */
-STATUS getIceCandidateStats(PRtcPeerConnection, PRtcIceCandidateStats);
+STATUS getIceCandidateStats(PRtcPeerConnection, PRtcIceCandidateStats, BOOL);
```

Comment:
our convention is to put in/out param as the last param. For example the createXXX apis.

Reply:
Done